tee-worker/ts-tests/README.md

```diff
@@ -0,0 +1,43 @@
+## Description
+
+ts-tests of tee-worker
+
+## Environment
+
+Node versions 16.x or 18.x, yarn v3
+
+## Installation
+
+`cd tee-worker/ts-tests`
+
+`yarn`
+
+## Type Generated
+
+Update parachain metadata: `yarn workspace parachain-api update-metadata`
```

Comment:
Should note that this requires a working runner on localhost:9944

Reply:
Following up on this: 

generating sidechain metadata requires that worker is running, doesn't it